### PR TITLE
HPCC-13216 Use relative paths based on exe current path to find resource...

### PR DESCRIPTION
**Pull request by rpastrana**
Using relative paths based from the current execute location to locate the componentfiles folder
Signed-off-by: rpastrana rodrigo.pastrana@lexisnexis.com

### CODE REVIEW

**Comment by rpastrana** [comment]
@RussWhitehead please review

**Comment by rpastrana** [comment]
@RussWhitehead please review

**Comment by rpastrana** [comment]
@richardkchapman this PR has been tested on Win and Lin successfully and code has been reviewed.

**Comment by richardkchapman** [comment]
@ghalliday Please sanity check

File: tools/esdlcmd/esdlcmd_common.hpp [hunk 1]
```diff
@@ -300,4 +303,27 @@ class EsdlHelperConvertCmd : public EsdlConvertCmd
     virtual void setTransformParams(IProperties *params )=0;
 };
 
+static bool getCurrentFolder(StringBuffer & path)
```
Comment (ghalliday) [inline tools/esdlcmd/esdlcmd_common.hpp]: minor: unusual to have static functions defined in header functions.  More normal to be declared inline.

**Comment by ghalliday** [comment]
@richardkchapman looks ok afaics.

**Pull request merged**